### PR TITLE
[FE-106] Correct progress bar data

### PR DESCRIPTION
**Pull request by AnCichocka**
- delete fake data
- little undefined clean up
https://oops2000.atlassian.net/browse/SCRUM-106

### CODE REVIEW

File: frontend/src/hooks/Students/useStudentsData.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export const useStudentsData = (editionId?: string) => {
             start: group.startTime,
             end: group.endTime,
           },
-          // TODO nullable
+          // TODO it shouldn't be nullable?
           teacher: {
```
Comment (AnCichocka) [inline frontend/src/hooks/Students/useStudentsData.tsx:36]: @Szyntos still valid question

Comment (AnCichocka) [reply frontend/src/hooks/Students/useStudentsData.tsx:36]:
```
  getStudentPoints(editionId: $editionId, studentId: $studentId) {
    user {
      firstName
      indexNumber
      nick
      secondName
      imageFile {
        fileId
      }
      userId
      userGroups {
        group {
          groupsId
          endTime
          groupName
          startTime
          weekday {
            weekdayId
            weekdayName
          }
          teacher {
            firstName
            secondName
          }
        }
      }
    }
    sumOfAll
    sumOfBonuses
    sumOfPurePoints
    subcategoryPoints {
      points {
        purePoints {
          value
          teacher {
            firstName
            secondName
          }
          createdAt
          updatedAt
        }
        partialBonusType {
          partialValue
          bonuses {
            award {
              awardName
            }
            points {
              createdAt
              updatedAt
              teacher {
                firstName
                secondName
              }
            }
          }
        }
      }
      subcategory {
        subcategoryName
        category {
          categoryId
          categoryName
        }
        maxPoints
      }
    }
  }
```

Comment (Szyntos) [reply frontend/src/hooks/Students/useStudentsData.tsx:36]: it shouldn't be nullable on main

FYI If custom query uses not custom types (e.g. group) the fact that it is a custom type doesn't change a thing

Comment (AnCichocka) [reply frontend/src/hooks/Students/useStudentsData.tsx:36]: I have main merged and it is nullable

Comment (AnCichocka) [reply frontend/src/hooks/Students/useStudentsData.tsx:36]: I have main merged and it still seems like it's nullable

Comment (AnCichocka) [reply frontend/src/hooks/Students/useStudentsData.tsx:36]: @Szyntos, I guess we both agreed to ignore it for now? If so, this PR is ready

Comment (Szyntos) [reply frontend/src/hooks/Students/useStudentsData.tsx:36]: Yes, i've tried to fix it, didn't work, soooooo

**Review by Szyntos — APPROVED** [review]

**Pull request merged**